src/app-layout/__integ__/runtime-drawers.test.ts

```diff
@@ -38,6 +39,26 @@ for (const visualRefresh of [true, false]) {
       })
     );
 
+    test.only(
+      'should resize equally with tools or drawers',
+      setupTest(async page => {
+        await page.setWindowSize({ ...viewports.desktop, width: 1800 });
+        await page.click(wrapper.findToolsToggle().toSelector());
+        await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
+        await page.click(wrapper.findSplitPanel().findPreferencesButton().toSelector());
+        const tile = createWrapper().findModal().findContent().findTiles().findItemByValue('side');
+        await page.click(tile.toSelector());
+        await page.click('button=Confirm');
```

Comment:
I would be more happy to add support for the query parameter on this page as well, rather than doing it manually